packrat/src/main/java/com/github/reflectoring/infiniboard/packrat/dashboard/DashboardConfig.java

```diff
@@ -1,16 +1,23 @@
 package com.github.reflectoring.infiniboard.packrat.dashboard;
 
+import java.util.List;
+
 import org.springframework.data.annotation.Id;
 
+import com.github.reflectoring.infiniboard.packrat.widget.WidgetConfig;
+
 public class DashboardConfig {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)